.github/workflows/release_droid_upload_github_release_assets.yml

```diff
@@ -12,39 +12,14 @@ jobs:
     uses: ./.github/workflows/check_precommit.yaml
 
   integration_tests:
-    strategy:
-      fail-fast: false
-      matrix:
-        python-version: [3.8]
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-        with:
-          python-version: ${{ matrix.python-version }}
-      - name: Install Poetry
-        uses: abatilo/actions-poetry@v2.1.6
-        with:
-          poetry-version: 1.2.1
-
-      - name: Poetry install
-        run: poetry install
-      - name: Poetry build
-        run: poetry build
-
-      - name: Checkout test environment
-        run: git clone https://github.com/exasol/integration-test-docker-environment.git
-        working-directory: ..
-      - name: Spawn EXASOL environemnt
-        run: ./start-test-env spawn-test-environment --environment-name test --database-port-forward 8888 --bucketfs-port-forward 6666 --db-mem-size 4GB
-        working-directory: ../integration-test-docker-environment
+    uses: ./.github/workflows/pytest.yaml
 
-      - name: Poetry run pytest integration tests
-        run: poetry run pytest tests
+  release-documentation:
+    needs: [integration_tests, check-release]
+    uses: ./.github/workflows/create_release_docu.yaml
 
   upload:
-    needs: [integration_tests, check-release]
+    needs: [publish-release-documentation]
```

Comment:
Is this really correct? The jobname in line 17 is "release-documentation".

Reply:
good catch